e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -0,0 +1,512 @@
+
+        JSONArray output = json.getJSONArray("output");
+        JSONObject outputObject = output.getJSONObject(0);
+        String url = outputObject.getString("url");
+        Assert.assertEquals(url, AB2D_API_URL + "Job/" + jobUuid + "/file/S0001.ndjson");
+        String type = outputObject.getString("type");
+        Assert.assertEquals(type, "null");
+    }
+
+    private void verifyJsonFromfileDownload(String fileContent) throws JSONException {
+        // Some of the data that is returned will be variable and will change from request to request, so not every
+        // JSON object can be verified
+        final JSONObject fileJson = new JSONObject(fileContent);
+        Assert.assertEquals(8, fileJson.length());
+        Assert.assertEquals("ExplanationOfBenefit", fileJson.getString("resourceType"));
+        Assert.assertEquals(0, fileJson.getInt("precedence"));
+        String carrierString = fileJson.getString("id");
+        Assert.assertTrue(carrierString.startsWith("carrier"));
+        final JSONObject patientJson = fileJson.getJSONObject("patient");
+        String referenceString = patientJson.getString("reference");
+        Assert.assertTrue(referenceString.startsWith("Patient"));
+        final JSONObject typeJson = fileJson.getJSONObject("type");
+        Assert.assertNotNull(typeJson);
+        final JSONArray codingJson = typeJson.getJSONArray("coding");
+        Assert.assertNotNull(codingJson);
+        Assert.assertEquals(4, codingJson.length());
+        final JSONArray identifierJson = fileJson.getJSONArray("identifier");
+        Assert.assertNotNull(identifierJson);
+        Assert.assertEquals(2, identifierJson.length());
+        final JSONArray diagnosisJson = fileJson.getJSONArray("diagnosis");
+        Assert.assertNotNull(diagnosisJson);
+        final JSONArray itemJson = fileJson.getJSONArray("item");
+        Assert.assertNotNull(itemJson);
+    }
+
+    @Test
+    public void runSystemWideExport() throws IOException, InterruptedException, JSONException {
+        String contractNumber = "S0001";
+        HttpResponse<String> exportResponse = exportRequest();
+
+        Assert.assertEquals(202, exportResponse.statusCode());
+        List<String> contentLocationList = exportResponse.headers().map().get("content-location");
+
+        HttpResponse<String> secondExportResponse = exportRequest();
+        Assert.assertEquals(429, secondExportResponse.statusCode());
+
+        HttpResponse<String> statusResponse = statusRequest(contentLocationList.iterator().next());
+
+        Assert.assertEquals(202, statusResponse.statusCode());
+        List<String> retryAfterList = statusResponse.headers().map().get("retry-after");
+        Assert.assertEquals(retryAfterList.iterator().next(), String.valueOf(DELAY));
+        List<String> xProgressList = statusResponse.headers().map().get("x-progress");
+        Assert.assertEquals(xProgressList.iterator().next(), "0% complete");
+
```

Comment:
There is no way we can expect a certain completion percentage value. Let's just verify the header matches a regexp.